emsdk.py

```diff
@@ -1280,7 +1280,17 @@ def emscripten_npm_install(tool, directory):
   npm = os.path.join(node_path, 'npm' + ('.cmd' if WINDOWS else ''))
   env = os.environ.copy()
   env["PATH"] = node_path + os.pathsep + env["PATH"]
-  subprocess.check_call([npm, 'install', '--production', '--no-audit'], cwd=directory, env=env)
+  print('Running post-install step: npm install ...')
+  try:
+    subprocess.check_output(
+        [npm, 'install', '--production', '--no-audit'],
+        cwd=directory, stderr=subprocess.STDOUT, env=env,
+        universal_newlines=True)
+  except subprocess.CalledProcessError as e:
+    print('Error running %s:\n%s' % (e.cmd, e.output))
+    return False
```

Comment:
before this PR the check_call threw, but now we will return false?

Reply:
Its better to return false.  Makes for cleaner error messages.

Reply:
Was the thrown error caught higher up? Or is the false turned into an error or exit? (Otherwise it used to halt, but no longer does?)

Reply:
Yeah the False return value is handled by the caller as an error.